component.yaml

```
@@ -0,0 +1,16 @@
+platform: Common
+buildPlatform:
+displayName: "Change Java Version"
+description: "Switch the current Java version to the selected one."
+inputs:
+  - key: "AC_SELECTED_JAVA_VERSION"
+    defaultValue: "17"
+    isRequired: true
+    title: "Java Version"
+    editorType: select
+    options: "8,11,17,21"
+    description: "Select the Java version to switch to."
```

```suggestion
    description: "Change the Java version in the environment to the selected Java version."
```
For more clear understand we can change the description like that.

---

@boztopuz I deliberately did not extend this explanation, I kept it simple and understandable.
Did you think it was incomprehensible?
If so, I can improve it further, but I did not find the suggested change correct. The reasons are:

### `Change`:
This improvement will come with the following default variable. Here, we want it to do "Select", not "Change"

### `in the environment`:
This component does not only change the environment, it also changes the system Java version.

### `to the selected Java version`
I avoided repeating words in the same sentence, so I deliberately did not add this, but I will improve it a little more.

How to use the following:
"Select the Java version to switch to. If the selected Java version is not found in the runner, it will throw an error."

---

The last change you mentioned is okey